index.js

```diff
@@ -281,13 +271,11 @@ function loadConfig(program) {
     }
 
     utils.log(`${config.package} started on ${config.hostname} (${config.ip})`)
-    if (config.platform && config.platform.startsWith('k8s')) { k8s.init() }
     if (config.userAgent) {
       config.DEFAULT_REQ_HEADERS['user-agent'] = config.userAgent
       config.DEFAULT_REQ_HEADERS_GZIP['user-agent'] = config.userAgent
     }
 
-    debug('connecting to log server')
```

Comment:
I'd say let's keep this debug statement here - unrelated to `K8s` support and if we remove this from here, we need to add something similar to `start` functionality showing that the execution reached this point to make connection with the server

Reply:
sure.  I just removed it because it's not accurate.  "connect" makes it seem persistent, but it's just a POST endpoint.  I can keep it, but re-word it.